.github/workflows/ci.yaml

```diff
@@ -22,6 +22,10 @@ jobs:
         run: make project-tools
       - name: Lint
         if: ${{ matrix.go-version == '1.19' || matrix.go-version == '1.20' }}
-        run: make lint
+        run: |
```

Comment:
I think it would be better here if we executed `make lint` because other steps might be added later besides the `sync-tidy` and `golangci-lint`. 

But I see the point of failing early after the sync tidy. Maybe moving the `make sync-tidy` step inside the `check-sync-tidy` script and then executing `make lint`, with the drawback of doing the sync twice? :thinking: